### PR TITLE
Fix filter logic error: redundant produce (92X)

### PR DESCRIPTION
**Pull request by afiqaize**
Hi,

This commit is to fix the logic error in the filter. Otherwise it gives [1]. Probably escaped attention as this filter isn't in use in the current menu.

Cheers,
Afiq

[1]
----- Begin Fatal Exception 13-Jun-2017 17:32:55 CEST-----------------------
An exception of category 'LogicError' occurred while
   [0] Constructing the EventProcessor
   [1] Constructing module: class=HLTEgammaGenericQuadraticEtaFilter label='hltEle27WPTight1HEFilter'
Exception Message:
Duplicate Product The produced product Branch Type = Event
Process Name = MYHLT
ModuleLabel = hltEle27WPTight1HEFilter
Branch ID = 1294368331
Class Name = trigger::TriggerFilterObjectWithRefs
Friendly Class Name = triggerTriggerFilterObjectWithRefs
Product Instance Name =
 is registered more than once.
Please remove the redundant 'produces' call(s).
----- End Fatal Exception -------------------------------------------------

### CODE REVIEW

**Comment by Martin-Grunewald** [comment]
please test

File: HLTrigger/Egamma/src/HLTEgammaGenericQuadraticEtaFilter.cc [hunk 1]
```diff
@@ -91,7 +91,7 @@ HLTEgammaGenericQuadraticEtaFilter::HLTEgammaGenericQuadraticEtaFilter(const edm
   }
 
   //register your products
-  produces<trigger::TriggerFilterObjectWithRefs>();
+  //produces<trigger::TriggerFilterObjectWithRefs>();
```
Comment (fwyzard) [inline HLTrigger/Egamma/src/HLTEgammaGenericQuadraticEtaFilter.cc]: please remove the line instead of commenting it

**Comment by fwyzard** [comment]
@cmsbuild please test

**Comment by Martin-Grunewald** [comment]
please test

**Comment by fwyzard** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**